tasks/pipeline.py

```diff
@@ -9,7 +9,17 @@
 
 @task
 def trigger(ctx, git_ref="master", release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"):
-    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch)
+    pipeline_id = trigger_agent_pipeline(git_ref, release_version_6, release_version_7, repo_branch, deploy=True)
+    wait_for_pipeline("DataDog/datadog-agent", pipeline_id)
+
+
+@task
+def run_with_all_tests(
```

Comment:
Nit: I personally like
```suggestion
def run_all_tests(
```
better, since it is shorter.